src/sbl_filing_api/entities/repos/submission_repo.py

```diff
@@ -128,11 +128,13 @@ async def upsert_filing(session: AsyncSession, filing: FilingDTO) -> FilingDAO:
     return await upsert_helper(session, filing, FilingDAO)
 
 
-async def create_new_filing(session: AsyncSession, lei: str, filing_period: str) -> FilingDAO:
+async def create_new_filing(
+    session: AsyncSession, lei: str, filing_period: str, institution_snapshot_id: str = None
+) -> FilingDAO:
     new_filing = FilingDAO(
         filing_period=filing_period,
         lei=lei,
-        institution_snapshot_id="v1",  # need story to retrieve this from user-fi I believe
+        institution_snapshot_id=institution_snapshot_id,  # need story to retrieve this from user-fi I believe
```

Comment:
We wouldn't be passing this in, potentially, on a new filing.  It's something the frontend would be setting with our /institution_snapshot_id endpoint after the filing has already been created.  But this still needs to be discussed.  Probably need a frontend and backend meeting, 15-30 mins tops, to decide how to proceed with this field.  It may be that in MVP we just keep the default, or we could do it this way which would I think slightly alter the frontend flow to call the POST to create the filing after the user has set institution data.

Either way, probably could remove the # need story to retrieve this... comment ;)